src/styles/content.scss

```diff
@@ -0,0 +1,99 @@
+$blue: #00acdc;
+$main-accent: #060b35;
+$secondary: #616070;
+
+.main {
+  margin: 0 auto;
+  padding: 50px 0;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  width: 1000px;
```

Comment:
now you hardcode the width, but it should be calculated dynamically taking into account the card's width, the max number of cards in a row, and the distance between them